samples/mpu/mem_domain_apis_test/src/main.c

```diff
@@ -26,13 +26,24 @@ struct k_thread app_thread_id[3];
 struct k_mem_domain app_domain[2];
 
 /* the start address of the MPU region needs to align with its size */
+#ifdef CONFIG_ARM
```

Comment:
we need to get rid of the hardcoded values here and have macros for users to do this

Reply:
still need to fix this.
what is this value supposed to represent? stack buffer guard size?

Reply:
yep it is the same. I can change it to use that stack guard macro. But just thinking here, the name of the macro might cause confusion.